### DISCUSSION TITLE
How to find and replace new lines?

### DISCUSSION THREAD

**Discussion by ghost** [geany/geany#3955] [General]
I typed `\n`, tried both regular expressions with multi-line matching and escape sequences. None of them worked.

I'm using Geany on Windows via PortableApps.

p/s: The file itself is UTF16-LE (with BOM), with Windows line endings (CRLF).

**Comment by elextr** [comment]
What exactly are you attempting to do?  What are you replacing the newline characters to?  Finding `\n` with regex and multi line settings as you used and replacing it to something silly like `a` works for me.

**Reply by ghost** [reply]
Nothing? I want to join the text into just one line.
